src/components/GameView/GameMenu.vue

```diff
@@ -145,24 +148,24 @@ export default {
     },
     async concede() {
       this.loading = true;
-      await this.$store.dispatch('requestConcede');
+      await this.gameStore.requestConcede();
       this.shownDialog = '';
     },
     async requestStalemate() {
       this.loading = true;
       try {
-        await this.$store.dispatch('requestStalemate');
-        this.$store.commit('setWaitingForOpponentToStalemate', true);
+        await this.gameStore.requestStalemate();
+        this.gameStore.setWaitingForOpponentToStalemate(true);
```

Comment:
This looks like another method that just sets a single value; if so, let's remove it
```suggestion
        this.gameStore.waitingForOpponentToStalemate = true;
```